NASA_API.pyw

```diff
@@ -20,7 +21,8 @@ def main():
     image_data = requests.get(hdurl)
 
     # creating path and name of image
-    image_path = 'E:/Theseus/Pictures/NASA_APOD/%s_NASA_APOD.png' %date
+    current_dir = str(pathlib.Path(__file__).parent.resolve())
+    image_path = current_dir + '/APOD_images/%s_NASA_APOD.png' %date
```

Comment:
There is a problem here...

What happens if the directory 'APOD_images' doesn't exist? It will throw an error.

You'll have to add logic to create the directory if doesn't exist.

Reply:
I was thinking of having an "APOD_images" directory on the project files so that it's added as soon as the project is cloned, do you think it's best to have the program check for one instead and create it if it doesn't exist?

Reply:
Yes, I don't think is a good practice to add empty directories to git, is probably better for the program to create it at run-time.

Also, my next feature request (after you finish this one) is going to be that the user might want to save the pictures in a different folder, besides the default one. So at that point, you still might want to create the default if there is no other folder specified by the user.